src/Randomizer.App/ViewModels/GeneralOptions.cs

```diff
@@ -67,7 +82,7 @@ public static IEnumerable<string> QuickLaunchOptions
 
         public bool TrackerSpoilersEnabled { get; set; }
 
-        public bool AutoTrackerAutoStart { get; set; }
+        public int AutoTrackerConnectorType { get; set; } = (int)EmulatorConnectorType.None;
```

Comment:
Why is this an int?

Reply:
Because unfortunately, last I knew, C# combo boxes don't work super well being based off of enums with descriptions, so one of the ways around that was to select the descriptions as strings and use them as the value and bind the selected index back, which is then casted by to the same enum used to build the combo box.

I could look into creating an extension or type converter for the enum if that would be preferred.